### PR TITLE
This a clone of the Newsweek Magazine. 

### PR DESCRIPTION
**Pull request by Arinpe**
I worked alone on this project and I was able to achieve the clone of the Newsweek page using HTML, CSS and Bootstrap. I resolve the issues I was asked to do.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>NewsWeek</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <link rel="stylesheet" type="text/css" href="/style.css">
```
Comment (umairarshadbutt) [inline index.html]: Please update it as I mentioned in the review

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @Arinpe Arinpe
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Fix the Linters errors
- [x] The project is working fine locally but not online. Please Correct it. 
Instead of  `<link rel="stylesheet" type="text/css" href="/style.css">`
It should be `<link rel="stylesheet" type="text/css" href="./style.css">`
- [x] Remove SVG from HTML file and save them in assets
- [x] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/97987394-06073880-1dfd-11eb-8067-32d191b780c2.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/) ([screenshot](https://user-images.githubusercontent.com/29689273/94326203-fa358480-ffbb-11ea-81ec-b2704fb93212.png) from Pathwright)



### Navbar
- [x] Logo is missing in the navbar, please add the logo with the navigation link

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### Navbar (small screen)
- [x] Add the user icon
- [x] Add the Logo with navigation link

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### Header
- [x] Article count should be one in small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### Missing Image
- [x] **IN THE MAGAZINE** image is missing, please add the image

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### IN THE MAGAZINE section
- [ ] Please add the spacing using margin on other images as I explained in images

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### SUBSCRIBE
- [x] Update the design as source landing page

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|


### NEWSLETTER
- [x] Update the design as source landing page

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|

### Footer (Small screen)
- [x] Please add the Logo with the navigation link
- [x] Manage the social media icons and other links like source page

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/Arinpe/Bootstrap/5c932ee480a2953d6db4f25fbe51d3be52b49a3c/index.html)|
|:---:|:---:|
|source-page|clone|













##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Arinpe  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Your online link is not working, so please update it, since you are using githack, you can copy the `URL for development` which update itself whenever you push new codes, the one you copied is for production, consequently it didn't get updated when you changed the url for your styles link
- [x] Make sure that all of the images use relative paths (the paths which start with a dot) so that they will work on any environment, currently most of your images are being missed online (not shown online) on the page because you are using wrong path pattern. Example: instead of writting `` it should be ``
- [x] The mobile version of the clone should fill the whole screen, currently it's only taking a small part of the screen and stay skeezed on the left side of the screen
- [x] The newsletter section should have the image on mobile screen, currently it's getting hidden
- [x] Remember to use a red newsweek logo in the footer, currently you are using the black one
- [x] I noticed that you are using a lot of `display: grid` in your css file, as this project is focusing mainly on bootstrap, you should take advantage of that, try to use bootstrap utility classes instead of using grid or flex. [This link might help you](https://getbootstrap.com/docs/4.3/layout/grid/)
- [x] The name of your pull request should be descriptive
# [Optional changes] :large_blue_circle: 
- [ ] Your commit messages should be in imperative mode or present tense, you should have used `Start the mark up on more stories` instead of `started the mark up on more stories`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by Arinpe** [comment]
The mobile version of the clone should fill the whole screen, currently, it's only taking a small part of the screen and stay squeezed on the left side of the screen. This part is displaying well on Mozilla firefox, I don't understand why it is not displaying well on chrome

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes  ♲
Hi @Arinpe, 👏 You did very well on this project. ✨ Big-up for successfully replicating the Newsweek home page. Most requirements have done 👌, Kindly fix the following issues.

[REQUIRED]

- [ ] Update the live link in the readme file.
       You can use the following link [https://raw.githack.com/Arinpe/Bootstrap/newsweek/index.html](https://raw.githack.com/Arinpe/Bootstrap/newsweek/index.html)
- [ ] Set the avatars circle in the "The Debate" section.

- [ ] Set appropriate height and width for the Newsweek logo at the bottom of the page.
- [ ] Set the header position fixed.
- [ ] Fix the section title position on the editors pick sections
editors original
editors-yours

- [ ] Fix the height of the cards in the "IN THE MAGAZINE" section. 
in the magazine original
in the magazine



Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/)

**Comment by Arinpe** [comment]
- All the required changes have been fixed.

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### changes required
Hi guys, great job with the project so far. there are still some changes to be made.

### required

- [ ] Please adjust the position of the logo
aasdasd

- [ ] this story disappears in small large screed and doesn't look good in small
sadsasd

- [ ] please fix the spacings in the debate section.

 asdasdadada  

- [ ]  the image size is too big
asgagaga

- [ ] there is a whole section missing after the editor's picks section

---

please fix all of these issues and ask for another code review

**Comment by Arinpe** [comment]
- The required section has been added. 
- All other changes have been fixe.

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @Arinpe  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, with all of changes that were required to make you finally achieved an outstanding clone congratulations on the good work you have done so far. 
:+1: I really wanted to approve this project but there is a small issue to fix before the approval,

# [Required Changes]
- [x] The image on Subscribe section is overlapping with text on screens between (768px - 999px), please add spacing between them as the original page is, study the screenshot below to understand what I mean

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-11-05 19-03-41|Screenshot from 2020-11-05 19-05-48|
:point_right: Could you see the difference :question: :question: :question: 

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes requested 
**Hi guys**
Nice job implementing the required changes, but there are still some changes to be made:

- please fix all the linters issues before asking for another code review

**Comment by mimipeshy** [comment]
## STATUS: APPROVED :champagne:  :dancers: 

Hi @Arinpe  :wave:  :wave: , great job on working on all comments given by the TSE's, I know it's a lot but great job for working on all of them. I, therefore, have to approve your project :partying_face:  . Go ahead and merge, all the best in the next project!

Regards
Peris Ndanu

**Pull request merged**